### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mishrasamiksha**
Corrected three words typo error

### CODE REVIEW

**Review by eriknordmark — CHANGES_REQUESTED** [review]
Thank you for fixing this.
But you need to use git commit --signoff to add a signoff line to the commit. That is required by LF-Edge policy. See git log what those message look like.

You can add that using e.g., --reedit-message but if you do it manually you need to make sure it matches the email address associated with your github account.

File: README.md [hunk 1]
```diff
@@ -515,7 +515,7 @@ You may be wondering why do we have a container-based architecture for a Xen-cen
 1. Set up the filesystem root using [containerd](https://containerd.io)
 1. Launch the domU using Xen via `xl`
 
-In addition to that, while we plan to build a fully disagregated system (with even device drivers running in their separate domains) right now we are just getting started and having containers as a first step towards full disagreagation seems like a very convenient stepping stone.
+In addition to that, while we plan to build a fully disaggregated system (with even device drivers running in their separate domains) right now we are just getting started and having containers as a first step towards full disaggreagation seems like a very convenient stepping stone.
```
Comment (milan-zededa) [inline README.md:518]: disaggreagation -> disaggregation

one hell of a word :)

**Comment by rene** [comment]
@mishrasamiksha , thanks for the fix. I'd also recommend to squash these two commits into only one, besides addressing @eriknordmark's comments... changes are simple and in the same file, so no reasons to split into two commits...

**Comment by mishrasamiksha** [comment]
sure, i'll make a single commit of these

**Comment by mishrasamiksha** [comment]
can i close this and create a new PR ? with a single commit ?

**Comment by mishrasamiksha** [comment]
commit 7476e27fade58d14a0bcde589aec22fc19fb49ec (HEAD -> readme_changes)
Author: Samiksha Mishra <38784342+mishrasamiksha@users.noreply.github.com>
Date:   Mon Oct 30 13:01:07 2023 +0530

    Merged 2 commits Update README.md

**Comment by mishrasamiksha** [comment]
can you please check this now 
I guess this should work

**Comment by eriknordmark** [comment]
It doesn't look like you updated the PR,
If you squashed the commits (and added the signed line to the commit message), then you need to do a git push -f to update your branch and the PR.

**Comment by mishrasamiksha** [comment]
oh right
it is fine now. I can see it. 
thanks (from this i learnt to squish, all thanks to you)

**Comment by yamcodes** [comment]
I'm curious about requiring contributors to manually squash commits when GitHub offers "[Squash and merge](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#squash-and-merge-your-commits)" for committing PR's as one unit of work to `main`. The linked GitHub documentation explains the value of this feature:

> You can use squash and merge to create a more streamlined Git history in your repository. Work-in-progress commits are helpful when working on a feature branch, but they aren’t necessarily important to retain in the Git history. If you squash these commits into one commit while merging to the default branch, you can **retain the original changes with a clear Git history**.

Your [CONTRIBUTING.md](https://github.com/lf-edge/eve/blob/master/CONTRIBUTING.md) file requires enthusiastic contributors to handle this on their own behalf. It also states that it welcomes "**any and all types of contributions** that benefit the community at large, not just code contributions" and that it stands for "Community over Code".

Keeping the above in mind, is there a clear advantage to this strict approach over the more relaxed alternative of having Triage roles use 'Squash and merge' directly?

**Comment by milan-zededa** [comment]
> I'm curious about requiring contributors to manually squash commits when GitHub offers "[Squash and merge](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#squash-and-merge-your-commits)" for committing PR's as one unit of work to `main`. The linked GitHub documentation explains the value of this feature:
> 
> > You can use squash and merge to create a more streamlined Git history in your repository. Work-in-progress commits are helpful when working on a feature branch, but they aren’t necessarily important to retain in the Git history. If you squash these commits into one commit while merging to the default branch, you can **retain the original changes with a clear Git history**.
> 
> Your [CONTRIBUTING.md](https://github.com/lf-edge/eve/blob/master/CONTRIBUTING.md) file requires enthusiastic contributors to handle this on their own behalf. It also states that it welcomes "**any and all types of contributions** that benefit the community at large, not just code contributions" and that it stands for "Community over Code".
> 
> Keeping the above in mind, is there a clear advantage to this strict approach over the more relaxed alternative of having Triage roles use 'Squash and merge' directly?

Doing this automatically leads to ugly commit messages IMHO. Making little effort to keep the commit history clean and easily readable helps everyone who contributes and maintains this project, therefore it is as much appreciated as the commit content itself.
New "enthusiastic contributors" also benefit by learning a new useful skill instead of relying on some automation tool, which outside of Github might not be available.

**Comment by eriknordmark** [comment]
> > Keeping the above in mind, is there a clear advantage to this strict approach over the more relaxed alternative of having Triage roles use 'Squash and merge' directly?

In addition to getting clear commit messages there are plenty of cases where the developer intentionally wants to several commits in a PR when they each commit can be built independently (e.g., add header files, add code, add docs, add tests can be separate commits). It is the developer who is the expert on this and it is hard for the person doing the merge do second-guess whether the intent was to have separate commits or they should be all done as one commit.

**Comment by eriknordmark** [comment]
You might have missed my first comment about DCO, but this is an LF Edge policy to require this. See 
https://github.com/lf-edge/eve/pull/3545/checks?check_run_id=18215718215
for information on how to add the signoff.

**Comment by rene** [comment]
@mishrasamiksha , so as @eriknordmark said, now you just need to sign-off your commit, which can be done with the following command:

`git commit --amend -s` 

Also, take the opportunity to edit the commit message, I would recommend something like this:

README.md: Fix some typos

So at the end if you run `git log` in your branch, you should see something like this in your commit:

```
    README.md: Fix some typos
    
    Signed-off-by: <Your name> <your@email>
```

**Review by eriknordmark — APPROVED** [review]
LGTM but the commit needs to be ammended as @rene suggested,

**Comment by rene** [comment]
I'm closing this PR since its inactivity. Changes here are now handled in https://github.com/lf-edge/eve/pull/3622

**Pull request closed without merging**